libraries/AP_RangeFinder/AP_RangeFinder_SRF02I2C.h

```diff
@@ -0,0 +1,64 @@
+#pragma once
+
+#include "RangeFinder.h"
+#include "RangeFinder_Backend.h"
+#include <AP_HAL/I2CDevice.h>
+
+enum {
+    SRF02_I2C_DEFAULT_ADDRESS = 0x70,
+    // http://robot-electronics.co.uk/htm/srf02tech.htm Written measurement value
+    SRF02_I2C_MIN_DISTANCE = 18,  // Not 16cm.
+    SRF02_I2C_MAX_DISTANCE = 249, // Not 600cm.
```

Comment:
Where is this 600cm? I can only find 249cm in that page. Doesn't it return 0 for more than 249cm?

Reply:
I saw that "Range: 16 cm to 6 m." Is written at the following URL.

https://www.dfrobot.com/wiki/index.php/SRF02_Ultrasonic_sensor_(SKU:SEN0005)
<img width="773" alt="2017-12-22 23 49 37" src="https://user-images.githubusercontent.com/646194/34302104-d26a23de-e772-11e7-8efe-26120aa5fd7e.png">

Reply:
OK, that's not the manufacturer's page. But I saw that the manufacturer actually has the 6m in their product page, but then the spec page shows the 249cm max, weird.

Reply:
@OXINARF san..... Thank you.
I can see that SRF02 is shipped from multiple manufacturers.
I cease to limit by the measured value of SRF02.
I return false if the specification exceeds 600 cm.
I think that it is not good for drivers to use values not written in the specification.

https://acroname.com/products/r287-srf02?sku=R287-SRF02